src/main/java/globalquake/ui/stationselect/DeselectAllAction.java

```diff
@@ -13,6 +13,8 @@ public class DeselectAllAction extends AbstractAction {
     public DeselectAllAction(StationDatabaseManager stationDatabaseManager) {
         super("Deselect All");
         this.stationDatabaseManager=stationDatabaseManager;
+
+        putValue(SHORT_DESCRIPTION, "Deselect All Available Stations");
```

Comment:
It would be better to keep the descriptions consistent and put "Deselects" here same as in DeselectUnavailableAction